mysql_mimic/schema.py

```diff
@@ -28,9 +29,51 @@ class Column:
     catalog: Optional[str] = "def"
 
 
+class ColumnField(str, Enum):
+    TYPE = ("type",)
+    COMMENT = ("comment",)
+    DEFAULT = ("default",)
+    IS_NULLABLE = ("is_nullable",)
+
+
 def mapping_to_columns(schema: dict) -> List[Column]:
```

Comment:
We allow a mapping for simple use cases.
More complex use cases should just instantiate Column objects directly.
(that is, I think Minerva should just build the columns instead of this new intermediate format)